frontend/src/routes/Infrastructure/InfraEnvironments/Details/InfraEnvironmentDetailsPage.tsx

```diff
@@ -195,6 +249,20 @@ const InfraEnvironmentDetailsPage: React.FC<InfraEnvironmentDetailsPageProps> =
                 usedHostnames={usedHostnames}
                 isBMPlatform={isBMPlatform(infrastructures[0])}
             />
+            <AddBmcHostModal
+                infraEnv={infraEnv}
+                isOpen={isBmcModalOpen}
+                onClose={() => setBMCModalOpen(false)}
+                onCreateBMH={getOnCreateBMH(infraEnv)}
+                onSaveISOParams={getOnSaveISOParams(infraEnv)}
+                usedHostnames={usedHostnames}
+                isBMPlatform={isBMPlatform(infrastructures[0])}
+            />
+            <AddBmcHostYamlModal
+                isOpen={isBmcYamlModalOpen}
+                onClose={() => setBMCYamlModalOpen(false)}
+                onCreateBmcByYaml={importYaml()}
```

Comment:
the `importYaml` should not be execured, right ?

```suggestion
                onCreateBmcByYaml={importYaml}
```